test/test_run.py

```diff
@@ -364,7 +366,7 @@ def test_format_durations():
     msg = Run.format_durations(durations, 2)
     expected = textwrap.dedent("""\
     =========== ================
-     benchmark   total duration 
+     benchmark   total duration   # noqa W291, need it to get the proper table format
```

Comment:
@datapythonista  This table generated a W291 trailing whitespace error for flake8.  I comment on the line but now there is an AssertionError. How can I avoid those two errors?
<img width="704" alt="image" src="https://user-images.githubusercontent.com/13194519/158447909-152184d6-5216-4735-8a4a-3a726b62cec8.png">